packages/autocomplete-preset-algolia/src/requester/createRequester.ts

```diff
@@ -29,7 +29,7 @@ export type RequesterParams<THit> = {
   ): TransformedRequesterResponse<THit>;
 };
 
-type TransformResponseParams<THit> = {
+export type TransformResponseParams<THit> = {
```

Comment:
are we still using these exports? maybe not needed anymore

Reply:
Removed them. Didn't see any issues.